src/components/Pane/Pane.Props.ts

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { Pane } from './Pane';
+
+export interface IPaneProps extends React.Props<Pane> {
+  /**
+   * Whether the pane is displayed.
+   * @default false
+   */
+  isOpen?: boolean;
```

Comment:
I don't think we want to replicate this bad behavior of Panel and Dialog. Controls should render by default. That is, it is very confusing to a consumer when you do something like:
```tsx
<Panel/>
```

...only to find nothing there and to have to figure out that you need `isOpen={ true }` just to make it show up.

Recommend this is renamed to `hidden` since there is an HTML `hidden` attribute already, and per the best practices, we want to have names that follow convention. (We're going through and scrubbing a bunch of these.)